packages/pixelboy/apps/journal/checkin/checkin.coffee

```diff
@@ -47,3 +49,34 @@ class PAA.PixelBoy.Apps.Journal.CheckIn extends AM.Component
 
       else
         FlowRouter.go 'journal'
+
+  getExternalImgUrl: (event) ->
+    url = event.currentTarget.value
+    externalImgUrl = null
+    imgPreview = $('.image-preview')
+    placeholder = 'http://placehold.it/350x150?text=your+art'
```

Comment:
Add a `TODO: Replace with internal art asset.` since we don't want to rely on outside services if we don't have to. But it's OK to prototype like this, just add a `TODO:` as well to say it's quickly done. We also use `HACK:` where you have to go around some limitations in an ugly way.